app/views/posts/show.html.erb

```diff
@@ -1,10 +1,27 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
-  </head>
-
-  <body>
-    <h1>Here is a specific post for a given user.</h1>
-  </body>
-</html>
+<div class="post_desc">
+
+  <div class="post_detail">
+    <div class="post_header">
+      <h4 class="post_title"><%= @post.title %> by <%= @post.author.name %></h4>
+      <p class="comments_likes">
+        Comments: <%= @post.comments_count %>, 
+        Likes: <%= @post.likes_count %>
+      </p>
+    </div>
+
+    <div class="post_body">
+      <%= @post.text %>
+    </div>
+  </div>
+
+  <div class="post_comments">
```

Comment:
- [ ] Please adjust your layout to function as the given wireframe:

yours|given
---|---
![image](https://github.com/binyamolango/Blog-App/assets/91486791/e448e138-1015-4715-a58f-0e3f4bbaed06)|![blog_single_post](https://github.com/binyamolango/Blog-App/assets/91486791/5f070f0d-f360-4f38-91a4-cc8df862c3ca)